### DISCUSSION TITLE
Assets location

### DISCUSSION THREAD

**Discussion by t3rry327** [Hexeption/MCP-Reborn#243] [Q&A]
The tasK :downloadAssets is from the mcp plugin itself, but when i execute :copyAssets, the assets should copy to run/assets , but this does not occur. Also, with this folder empty, minecraft is still working correctly, with all the textures. So the question is, where are the assets that minecraft uses on :runclient?

**Comment by Arbuzik-New** [comment]
Хуй знает братух

**Comment by yynngu** [comment]
In the directory of your project:  "projects/mcp/build/mcp/downloadClient/client.jar"

**Comment by mymolc** [comment]
Normally, yes—that would make sense—but in any case, I think launching Start.java is better than using the task.
